src/java/org/apache/cassandra/schema/CompressionParams.java

```diff
@@ -402,29 +475,55 @@ private static Integer parseChunkLength(String chLengthKB) throws ConfigurationE
      */
     private static int removeChunkLength(Map<String, String> options)
```

Comment:
this whole method could use some love on formatting, spaces, braces, no braces if one statement only ...